### DISCUSSION TITLE
Due to my github user profile, my application of student benefits was rejected

### DISCUSSION THREAD

**Discussion by yurisequoia** [community/community#27411] [GitHub Education]
Hi, I have submitted 12 applications for the student package, all were rejected, and from the fourth application, the only suggestion after rejection was "You are more likely to be verified if you have completed your GitHub user profile." The suggestion pointing to  [this website](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile), I modified the profile picture, name, bio, status in my user profile according to the steps on the website, and even made public my edu email, using my real name and photo (after trying to no avail, I changed it back to a virtual one), is still rejected, and the suggestion after rejection is still "You are more likely to be verified if you have completed your GitHub user profile." However, there is nothing to complete on my user profile. I tried looking for help in creating a new ticket, but was immediately closed by the GitHub Bot. I need help, thanks.

**Comment by fernandounger** [comment]
> Hi, I have submitted 12 applications for the student package, all were rejected, and from the fourth application, the only suggestion after rejection was "You are more likely to be verified if you have completed your GitHub user profile." The suggestion pointing to [this website](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile), I modified the profile picture, name, bio, status in my user profile according to the steps on the website, and even made public my edu email, using my real name and photo (after trying to no avail, I changed it back to a virtual one), is still rejected, and the suggestion after rejection is still "You are more likely to be verified if you have completed your GitHub user profile." However, there is nothing to complete on my user profile. I tried looking for help in creating a new ticket, but was immediately closed by the GitHub Bot. I need help, thanks.

What are you submitting to prove you are a student?

My applications were rejected 3 times because I was submitting something that did not confirm the eligibility of the document, but then they approved with another type of document.

**Reply by yurisequoia** [reply]
Thank you for your reply, as I am a student of a university in Chinese Mainland, I submitted the student status certificate applied from the Ministry of education of China (in Chinese, I translated an English version myself).But I don't think it was rejected because the submitted documents were not compliant.The latest times of rejection were within 1 minute after I submitted the application. The rejection prompt was "you are more likely to be verified if you have completed your GitHub user profile." , which doesn't seem to be related to the documentation I submitted.
<img width="334" alt="image" src="https://user-images.githubusercontent.com/39732771/181512957-1d08fba0-0d68-44ec-be42-13d510f973b2.png">

**Comment by FuryAsh21** [comment]
I faced the same problem when I was just using my edu mail. But then I tried manually using a gmail account. Selected my institution and then uploaded a student ID card as proof. And finally I got my student package. If you have more query, ask me.

**Reply by yurisequoia** [reply]
Thanks for your reply and suggestion. I tried to use gmail instead of edu mail, but the problem is still there, the suggestion after being rejected is still "You are more likely to be verified if you have completed your GitHub user profile."

**Reply by GFQAQ** [reply]
In China, try the Xuexin network

**Comment by Weigaa** [comment]
@YuriSequoia 我也遇到了一样的问题，我甚至将我的学生证、一卡通、成绩单、英文版学信网证明拼接成了一张照片并拍摄，仍然未能通过验证，也是遇到了类似需要补全profile的提示，可我的profile已经全部填写好了。
如果您解决了这个问题，希望能得到您的回复。

**Reply by zdaoguang** [reply]
请问有解决吗？谢谢

**Comment by LeCaToX** [comment]
I faced with the same problem as you 😭

**Reply by simaremare** [reply]
I got the same problem here. Have no idea what was wrong. The message is rather vague.

**Reply by LeCaToX** [reply]
I actually got accepted. I just use my personal email and tried others kind of documents (in my case I used school certificate)

**Reply by AlphaHasher** [reply]
@LeCaToX Did you just take a picture of your school certificate with a webcam? I'm asking because I've had massive issues with the resolution being too low to ready any form of ducemtation

**Reply by LeCaToX** [reply]
@Codingboy65 I acctually took a picture of my certificate with an iPad and then I opened the picture on my iPad and showed it to the web cam

**Comment by castlewong** [comment]
Same as you. And I try to complete the so-called Profile again and again, then, it worked.
The last thing I did was, set my in Profile-Contributions & Activity-Check "Make profile private and hide activity", try it, I mean, try every step the profile doc told you to.
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/24472312/185732977-274cdf16-003c-4f88-bd4e-5c04b74dbb59.png">

**Reply by AlphaHasher** [reply]
glad you were able to get through but still not working for me sadly, I'll keep trying different things

**Reply by castlewong** [reply]
Nope, I was just succeed in submitting, and minutes later I got email saying that I was denied.

**Comment by zdaoguang** [comment]
Good 
    
        
            
                
                    
                        
                    
                    
                        
                            昝道广
                            
                                    
                                    中国科学院
                                  
                                     - 
                                    学生
                            
                                
                                    13161691826
                                
                                
                                    ***@***.***
                                
                                
                                    北京市海淀区中关村南四街4号中国科学院软件研究所
                                
                        
                    
                
            
        
    
On 08/20/2022 14:49, castlewong wrote: 
Same as you. And I try to complete the so-called Profile again and again, then, it worked.
The last thing I did was, set my in Profile-Contributions & Activity-Check "Make profile private and hide activity", try it, I mean, try every step the profile doc told you to.


—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/orgs/community/discussions/27411#discussioncomment-3436071",
"url": "https://github.com/orgs/community/discussions/27411#discussioncomment-3436071",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Reply by castlewong** [reply]
Nope, 刚收到邮件，还是说我profile没完成，我真懵了，这到底咋回事？

**Comment by notscar** [comment]
This verification steps are just hell, School ID won't get accepted because by the AI says there's no name on it of my school, the school authenticated site says it's not trustworthy as not shot with camera and also says to complete github profile while i have done everything. Great service overall.

**Reply by castlewong** [reply]
Yeah, cannot figure out how to succeed.

**Comment by praksharma** [comment]
My application was accepted in first attempt in 2020. Now I made second attempt in 2022, after the previous one expired and was accepted again.

**Comment by dski97** [comment]
They make it extra hard for you to get in. Especially as a grad student doing an online degree.

**Comment by ghost** [comment]
>Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?

So how to solve this problem? My GitHub personal information is complete. 😅

**Reply by castlewong** [reply]
I do not know

**Reply by ghost** [reply]
Thanks for the reply, this seems to be just a GitHub bug. I tried several times separately, and after a few days, the re-authentication was successful.

**Comment by truemogician** [comment]
The verification process is really irritating. The pre-check did nothing but cause trouble for real students trying to get started with GitHub. It didn't use to be like this. GitHub should really learn from other companies like JetBrains, they manually review all applications that cannot be processed automatically, instead of arrogantly rejecting them with ambiguous instructions given by an incompetent pre-check AI.

**Comment by rsha256** [comment]
> Try applying again after making more [contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile#what-counts-as-a-contribution) to the GitHub Classroom assignment(s) created by your teacher.

I see the following but have made numerous contributions to github classroom assignments... what is going on lol

I'm also an in-person actual student... The support ticket is just a bot which just repeats the same things :|

**Reply by Meteor-Comet** [reply]
me too

**Comment by Meteor-Comet** [comment]
Try applying again after making more contributions to the GitHub Classroom assignment(s) created by your teacher.

**Comment by TenTraicion** [comment]
tried 7 times, with versity id card, transcript etc.. my versity dont provide student mail, but github asking me to add veristy mail as my email. i mean wth.. if they wont wanna give it just remove it from the options, making it hell is annoying ... also they want ma to complete my profile dunno what to add or what to remove. the AI is ...
